uber/templates/preregistration/preregbase.html

```diff
@@ -1,35 +1,135 @@
 {% extends "base.html" %}
-{% block backlink %}
-
-
-{% endblock %}
+{% block backlink %}{% endblock %}
 {% block header %}
-<link href="../static/theme/prereg.css" rel="stylesheet">
-<script type="text/javascript">
-    var checkCap = function () {
-        $.getJSON('../preregistration/check_prereg', function(check) {
-            if (check.force_refresh) {
-                location.reload();  // Reload the page to prevent registering after reg is closed
-            }
-        });
-    };
-    setInterval(checkCap, 1000 * 1 * 90);
+    <link href="../static/theme/prereg.css" rel="stylesheet">
+    <script type="text/javascript">
+        var PREREG_CHECK_INTERVAL = 15 * 60 * 1000;
+        var checkCap = function () {
+            $.getJSON('../preregistration/check_prereg', function (check) {
+                if (check.force_refresh) {
+                    location.reload();  // Reload the page to prevent registering after reg is closed
+                } else {
+                    setTimeout(checkCap, PREREG_CHECK_INTERVAL);
+                }
+            });
+        };
+        setTimeout(checkCap, PREREG_CHECK_INTERVAL);
 
-{% if c.COLLECT_FULL_ADDRESS %}
-$(document).ready(function() {
-    $("#international").hide()
-    setInternational()
-});
```

Comment:
This was just a style change; we're following more standard indentation and quoting conventions now.